package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@dojot/adminkafka": "0.0.1-alpha.9",
-    "@dojot/dojot-module": "^0.0.1-beta",
+    "@dojot/dojot-module": "0.0.1-beta.3",
```

Comment:
Better left this as it is - it is easier to just regenerate images than create a PR for all components with small version bumps.

Reply:
Ok!